client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -68,6 +86,58 @@ export default React.createClass( {
 		return ( this.props.domains || [ this.props.domain ] ).filter( domain => domain.currentUserCanManage );
 	},
 
+	wrongNSMappedDomains() {
+		debug( 'Rendering wrongNSMappedDomains' );
+		const wrongMappedDomains = this.getDomains().filter( domain =>
+			domain.type === domainTypes.MAPPED && ! domain.pointsToWpcom );
+
+		debug( 'Domains:', this.getDomains() );
```

Comment:
Do we really need to print all the site's domains here? It might make more sense if we put this line somewhere "higher" in this component, maybe even in the domains list, but I still think this is a bit overkill. Also, calling `this.getDomains()` isn't exactly free, since there's some filtering involved (not that it's costly either...) and it will be called every time this line is hit (rerenders, etc), whether debugging is enabled or not (at least that's what my testing with `CALYPSO_ENV=production make build` tells me).